notebooks/1_table_oriented.ipynb

```diff
@@ -0,0 +1,484 @@
+{
```

Comment:
<p>Rather than saying "methods are like functions", I think we should just say "methods are functions".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>